src/pages/components/button/usage.mdx

```diff
@@ -542,107 +539,805 @@ recommended action labels.
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not center align the label in a button, even if the button is wide.">
+
+![Do not center align the label in a button, even if the button is wide.](images/button_usage_6.png)
 
   </DoDont>
 </DoDontRow>
 
+#### Internationalization (RTL)
+
+For RTL (right-to-left) languages, the entire button is mirrored horizontally.
+The label is right-aligned and the icon is left-aligned.
+
+<Row>
+<Column colLg={8}>
+
+![Primary button without and with RTL.](images/button-usage-RTL.png)
+
+</Column>
+</Row>
+
+<Caption>Primary button without and with RTL.</Caption>
+
+### Overflow content
+
+When the button’s label is too long to fit within the available space of the
+button, the label should overflow and wrap to the second line. We do not
+recommend truncating a button label.
+
+As mentioned above, it is best practice to keep button label succinct and follow
+the &#123;verb&#125; + &#123;noun&#125; content formula.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do wrap the button label to a second line if there’s overflow content.">
+
+![Do wrap the button label to a second line if there’s overflow content.](images/button_usage_overflow_1.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not truncate the label of a button if there’s insufficient space inside the button.">
+
+![Do not truncate the label of a button if there’s insufficient space inside the button.](images/button_usage_overflow_2.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Further guidance
+
+For further content guidance, see Carbon’s
+[content guidelines](https://carbondesignsystem.com/guidelines/content/overview/).
+
 ## Universal behaviors
 
-### Primary button
+### Primary button focus
 
 The Primary button is the default action. When dialogs appear, the primary
 button typically takes focus. On a form, if focus is on a component that is not
 actionable with the `Enter` key, pressing `Enter` will activate the primary
 button.
 
+<Row>
+<Column colLg={8}>
+
+![Primary button takes default focus when there’s no other actionable component in a dialog.](images/button_usage_ub_focus.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Primary button takes default focus when there’s no other actionable component
+  in a dialog.
+</Caption>
+
+### States
+
+The default and interactive states of buttons should follow the
+button [Style](/components/button/style#color.mdx) tab guidelines.
+
 ### Interactions
 
 #### Mouse
 
 Users can trigger a button by clicking anywhere within the button container.
 
+<Row>
+<Column colLg={8}>
+
+![Mouse interaction for button.](images/button_usage_ub_interaction.png)
+
+</Column>
+</Row>
+
+<Caption>Mouse interaction for button.</Caption>
+
 #### Keyboard
 
 Users can trigger a button by pressing `Enter` or `Space` while the button has
 focus. For additional keyboard interactions, see the
-[accessibility tab](/components/button/accessibility).
+[Accessibility](/components/button/accessibility) tab.
+
+### Validation
```

Comment:
IMO I think we could cut this section since its basically a don't. It could be confusing to user.